src/domain/tokens/entities/schemas/token.schema.ts

```diff
@@ -6,7 +6,7 @@ export const tokenSchema: JSONSchemaType<Token> = {
   properties: {
     address: { type: 'string' },
     decimals: { type: 'number', nullable: true },
-    logoUri: { type: 'string', format: 'uri' },
```

Comment:
Why was this removed? Can it not have the `uri` format?

We should document this (in the PR for example).

Reply:
I was getting validation errors because of `logoUri` fields not starting with `http:://`. I had an example but it's not available right now due to staging service interruption 🤔 Maybe we should assume the `logoUri` is always following the URI format?

Reply:
That's a good question. We should check on this on the Transaction Service side. We can leave a comment either in the PR or in the codebase as to why it doesn't follow that format (depending on our findings).

Reply:
<img width="710" alt="Screenshot 2022-12-30 at 12 25 00" src="https://user-images.githubusercontent.com/21182069/210065331-ae83a621-ecc5-48c7-b707-ca54a7c12e50.png">

This is the response from the transaction service. We could investigate why the domain part missing in the `logoUri` URL. 

It may be related to the backup restoration these days, but in terms of the gateway it doesn't matter a lot I think, the fact it's it could not follow the URI format in some scenarios. IMHO we don't want to return a 500 from the gateway because the validation doesn't pass in those scenarios, what do you think? Do you think we need to error in the gateway response in that case?

Reply:
The current CGW does not return an error if the `logoUri` is not a a `uri` so let's remove the `format` as you did 👍